test/plugin/test_in_udp.rb

```diff
@@ -63,23 +80,33 @@ def test_time_format
         assert_equal_event_time(tests[i]['expected'], emits[i][1])
       }
     }
+
+    server.close
   end
 
   {
-    'none' => [
-      {'msg' => "tcptest1\n", 'expected' => 'tcptest1'},
-      {'msg' => "tcptest2\n", 'expected' => 'tcptest2'},
-    ],
-    'json' => [
-      {'msg' => {'k' => 123, 'message' => 'tcptest1'}.to_json + "\n", 'expected' => 'tcptest1'},
-      {'msg' => {'k' => 'tcptest2', 'message' => 456}.to_json + "\n", 'expected' => 456},
-    ],
-    '/^\\[(?<time>[^\\]]*)\\] (?<message>.*)/' => [
-      {'msg' => '[Sep 10 00:00:00] localhost: ' + 'x' * 100 + "\n", 'expected' => 'localhost: ' + 'x' * 100},
-      {'msg' => '[Sep 10 00:00:00] localhost: ' + 'x' * 1024 + "\n", 'expected' => 'localhost: ' + 'x' * 1024},
-    ]
+      'none' => [
+          {'msg' => "tcptest1\n", 'expected' => 'tcptest1'},
+          {'msg' => "tcptest2\n", 'expected' => 'tcptest2'},
+      ],
+      'json' => [
+          {'msg' => {'k' => 123, 'message' => 'tcptest1'}.to_json + "\n", 'expected' => 'tcptest1'},
+          {'msg' => {'k' => 'tcptest2', 'message' => 456}.to_json + "\n", 'expected' => 456},
+      ],
+      '/^\\[(?<time>[^\\]]*)\\] (?<message>.*)/' => [
+          {'msg' => '[Sep 10 00:00:00] localhost: ' + 'x' * 100 + "\n", 'expected' => 'localhost: ' + 'x' * 100},
+          {'msg' => '[Sep 10 00:00:00] localhost: ' + 'x' * 1024 + "\n", 'expected' => 'localhost: ' + 'x' * 1024},
+      ]
   }.each { |format, test_cases|
     define_method("test_msg_size_#{format[0] == '/' ? 'regexp' : format}") do
+      if Fluent.windows?
```

Comment:
I thought it's safer that we use original unused_port method in fluentd.
I fixed to use serverengine's api here. https://github.com/fluent/fluentd/pull/880/commits/75a07d24881a8087d6d0ea070b774869f562baf4